### ISSUE TITLE
Use Gradient colors for color contrast

### ISSUE THREAD

**Issue by HundredVisionsGuy** [HundredVisionsGuy/webcode-tk#20]
A student who adds a gradient to the body (global selector or whatever), fails color contrast even if `background-color` set a hex or rgb then sets a gradient on `background` property.

Example:
```
body {
    background-color: #001234;
    background: linear-gradient(#001234, #336699);
    color: #FFFFDE;
}
````
This will fail even if the background color and color meet color contrast goals.
 It should check the color off of any color stop in the gradient (not sure what to do about the alpha channel

**Comment by HundredVisionsGuy** [comment]
This is solved, and gradients are applied to background-image - I'm not sure about the background short-hand, but it does check for gradients, and then gets the lowest and highest contrast ratios, and if one or both fail, it fails.
